src/components/BlogForm/types.ts

```diff
@@ -1,7 +1,5 @@
-export type DataProps = {
+export type addFormData = {
```

Comment:
Call this something like `BlogFormType` to make it more specific to which form and that its a type